circuits/scripts/buildSnarks32BatchSize16.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")"
+cd ..
+mkdir -p params
+
+echo "Building batchUpdateStateTree_32_batch16"
+NODE_OPTIONS=--max-old-space-size=16384 node --stack-size=1073741 build/buildSnarks.js -i circom/prod/batchUpdateStateTree_32_batch16.circom -j params/batchUst32.r1cs -c params/batchUst32.c -y params/batchUst32.sym -p params/batchUstPk32.json -v params/batchUstVk32.json -s params/BatchUpdateStateTreeVerifier32.sol -vs BatchUpdateStateTreeVerifier32 -pr params/batchUst32.params -w params/batchUst32 -a params/batchUst32.wasm
```

Comment:
nit: change to 
`-s params/batchUpdateStateTree_32_batch16.sol`
so the names of smart contracts don't overlap